DICUI/Utilities/Converters.cs

```diff
@@ -271,20 +271,26 @@ public static List<string> KnownSystemAndMediaTypeToParameters(KnownSystem? sys,
                 return null;
             }
 
+            //TODO: adjust parameters according to paranoid mode
+
             // Now sort based on disc type
             List<string> parameters = new List<string>();
             switch (type)
             {
                 case MediaType.CD:
-                    parameters.Add(DICFlags.C2Opcode); parameters.Add("20");
+                    parameters.Add(DICFlags.C2Opcode); parameters.Add(RereadAmountC2 > 0 ? Convert.ToString(RereadAmountC2) : "20");
```

Comment:
If you can have a "paranoid" case for all CDs as well, I'd like to add `DICFlags.SubchannelReadLevel, "2"`, which is the slower but more accurate subchannel read-ahead.